packages/cli/test/unit/commands/promote.test.ts

```diff
@@ -0,0 +1,380 @@
+import chalk from 'chalk';
+import { client } from '../../mocks/client';
+import { defaultProject, useProject } from '../../mocks/project';
+import { Request, Response } from 'express';
+import promote from '../../../src/commands/promote';
+import { LastAliasRequest } from '@vercel-internals/types';
+import { setupUnitFixture } from '../../helpers/setup-unit-fixture';
+import { useDeployment } from '../../mocks/deployment';
+import { useTeams } from '../../mocks/team';
+import { useUser } from '../../mocks/user';
+import sleep from '../../../src/util/sleep';
+
+jest.setTimeout(60000);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏comment about why

When increasing the test timeout, I think we should leave a comment about why we need the longer timeout.